designs/metrics.md

```diff
@@ -101,6 +102,10 @@ represents the number of occurrences that were less than that buckets value. Sin
 histograms are a poor choice for unbounded data. Summaries instead directly track quantiles. This involves more
 computation but works better for unbounded data.
 
+Summaries do have a flaw: they can not be aggregated over multiple metrics. An example of this problem can be found in 
+the metric for node termination latencies. If we want to track the termination latencies of all nodes and still be able
+to filter over provisioners, this must be instrumented as two seperate metric families.
```

Comment:
Just a suggestion on wording that pulls from Prometheus Docs. Feel free to use what you want. 
```suggestion
Prometheus `Summary` metric types compute running quantiles for a given metric for given label values. Unlike histograms, summaries cannot be aggregated since it can result in statistically non-sensical results. Theoretically, summaries can be tracked for many labels, but we would need a number of summary metrics equal to the size of the cartesian product of all the labels, which is not viable.

Thus, if we want to track the termination latencies of all nodes and still be able
to filter over provisioners, this must be instrumented as two separate metric families.
```